README.md

```diff
@@ -73,6 +73,24 @@ To install this package on unix like systems run the following commands from the
 Detailed installation instructions, as well as instructions for installing PyRadiomics on Windows are available in the 
 [documentation](http://pyradiomics.readthedocs.io/en/latest/installation.html).
 
+### Docker
+
+PyRadiomics contains three "flavors" of [Dockers](https://www.docker.com/).  The first is a command line installation of the `pyradiomics` and `pyradiomicsbatch` command lines.  The second is a [Jupyter notebook](http://jupyter.org/) with PyRadiomics pre-installed with example Notebooks.  The third is a Ubuntu 16 based Docker for command line and interactive use. To build the Dockers:
+
+    docker build -t radiomics/cli .
+    docker build -t radiomics/notebook -f Dockerfile.notebook .
+    docker build -t radiomics/cli2 -f Dockerfile.ubuntu .
```

Comment:
The image should probably be named differently, it is not clear what is the difference between `cli` and `cli2`